e2e/tests/ui-driven/src/create-flow.spec.ts

```diff
@@ -177,6 +179,54 @@ test.describe("Flow creation, publish and preview", () => {
     await expect(previewLink).toBeVisible();
   });
 
+  test("Can add an external portal", async ({
+    browser,
+  }: {
+    browser: Browser;
+  }) => {
+    const page = await createAuthenticatedSession({
+      browser,
+      userId: context.user!.id!,
+    });
+
+    await page.goto(`/${context.team.slug}`);
+
+    const editor = new PlaywrightEditor(page);
+
+    page.on("dialog", (dialog) =>
+      dialog.accept(externalPortalServiceProps.name),
+    );
+    await editor.addNewService();
+
+    // update context to allow new flow to be torn down
+    context.flows?.push({ ...externalPortalServiceProps });
+
+    await createQuestionWithOptions(
+      page,
+      editor.firstNode,
+      externalPortalFlowData.title,
+      externalPortalFlowData.answers,
+    );
+    await expect(editor.nodeList).toContainText([
+      externalPortalFlowData.title,
+      externalPortalFlowData.answers[0],
+      externalPortalFlowData.answers[1],
+    ]);
+
+    await publishService(page);
+    await turnServiceOnline(page);
+
+    navigateToService(page, serviceProps.slug);
+
+    await createExternalPortal(page, page.locator("li:nth-child(6)"));
```

Comment:
I wanted it to be added in the middle of the flow rather than after the Review and Send components, so we would test going into an external portal and exiting an external portal. If I added it at the end with `getNextNode` then we wouldn't have that exit scenario convered?